README.md

```diff
@@ -71,6 +79,18 @@ npm run serve
     npm run run:${PLATFORM}
     ```
 
-# Tech notes
-
--   Use interfaces to [define prop types](https://github.com/facebook/create-react-app/issues/8021) in typescript
+# Guidelines
+
+- JS & Typescript: 
+    - use destructuring
+    - use TS types & interfaces 
+    - small to medium (1 -> ~400 lines) components
+    - reusable components should be within `src/components`
+    - reducers/actions/selectors should live alongside and be scoped the the domain (possible domains: room/call, dashboard/stats, group)
+    - actions,reducers, props should be typed
+- Libraries:
+    - keep the amount of libraries to a minimum to have the lower possible bundle size
+    - form: Formik, Yup
```

Comment:
you can add `material-formik` here maybe + `material-ui` for other UI components